packages/components/src/menu-group/index.js

```diff
@@ -19,6 +19,10 @@ export function MenuGroup( {
 	className = '',
 	instanceId,
 	label,
+	role = 'menu',
+	labelRole = ( role === 'group' ) ? { role: 'presentation' } : {},
+	useEventToOffset = true,
+	isScreenReaderLabel = false,
```

Comment:
not sure we need to visually hide the group "label", as groups with only one item shouldn't be groups (see below)